modules/imagetools/src/main/ts/ephox/imagetools/proxy/Errors.ts

```diff
@@ -19,8 +20,8 @@ const traverseJson = (json: any, path: string[]): Optional<any> => {
   return Optional.from(value);
 };
 
-const isServiceErrorCode = (code: number) =>
-  code === 400 || code === 403 || code === 500;
+const isServiceErrorCode = (code: number, blob: Blob | null): blob is Blob =>
```

Comment:
I see why this works and is named as it is but it feels that the function `isServiceErrorCode` shouldn't be returning whether `blob` is a `Blob`. I'm not sure what'd be a better solution though. `isServiceErrorCodeAndBlobIsBlob` doesn't seem much better.

Reply:
Being a service error implies that we have a valid blob, otherwise it wouldn't be a service-based error.

Reply:
I can't think of a name that makes anymore sense than this one either.

Reply:
I suppose it'll have to work. Just hoped there was something better.